libraries/eosiolib/multi_index.hpp

```diff
@@ -207,10 +207,12 @@ struct indexed_by {
  *  EOSIO Multi-Index table also supports up to 16 secondary indices. The type of the secondary indices could be any of:
  *  - uint64_t
  *  - uint128_t
- *  - uint256_t
  *  - double
  *  - long double
  *
+ *  `uint256_t` is not support for use as a secondary index, instead use the following
```

Comment:
Might be better not to mention `uint256_t`, since that type isn't defined anywhere